conf/application.conf

```diff
@@ -83,7 +83,7 @@ microservice {
             host = localhost
             port = 2003
             prefix = play.${appName}.
-            enabled = true
+            enabled = false
```

Comment:
Is turning off graphite by default intentional?

Have we checked it is turned on in the env configs?